Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -0,0 +1,660 @@
+# Import Fabric Install Utilities
+$fabricInstallUtilities = ".\Fabric-Install-Utilities.psm1"
+if (!(Test-Path $fabricInstallUtilities -PathType Leaf)) {
+    Write-DosMessage -Level "Warning" -Message "Could not find fabric install utilities. Manually downloading and installing"
+    Invoke-WebRequest -Uri https://raw.githubusercontent.com/HealthCatalyst/InstallScripts/master/common/Fabric-Install-Utilities.psm1 -Headers @{"Cache-Control" = "no-cache"} -OutFile $fabricInstallUtilities
+}
+Import-Module -Name $fabricInstallUtilities -Force
```

Comment:
I thought I commented here. In most of the places like above and below you are using force for import-module. Is it possible to have the 10 folder level version nesting issue, like with Install-Module when using force. Since all versions are saved locally.

Reply:
You did comment, but on the dos install utilities one which I updated so github obsoleted the conversation. We aren't at risk for that with the fabric install utilities since it is simply a file w/ no folder structure that we download, then import.